parent/main/resources/checkstyle.xml

```diff
@@ -36,13 +36,6 @@
       <!-- Checks for imports                              -->
       <!-- See http://checkstyle.sf.net/config_import.html -->
       <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
-      <module name="ImportOrder">
-         <property name="groups" value="java,javax,*"/>
-         <property name="ordered" value="true"/>
-         <property name="separated" value="false"/>
-         <property name="option" value="bottom"/>
-         <property name="sortStaticImportsAlphabetically" value="true"/>
-      </module>
```

Comment:
It was driving me crazy :) do we need to check for import order? the rest of the checkstyle plugin is fine (The problem is that intelliJ Idea is ordering imports differently than the plugin, and I don't see the option in Idea settings to change it)